### DISCUSSION TITLE
Import and Export Macros

### DISCUSSION THREAD

**Discussion by turtlewalker** [WarmUpTill/SceneSwitcher#482] [Ideas]
Hello! I see that there is a way to import/export settings. Is it possible to do the same with the actual macros? It would be nice for creating a lot of macros to be able to edit them in a text or csv doc and then import them.

**Comment by WarmUpTill** [comment]
At the moment no separate functionality to just import / export macros does exist.
But the import / export functionality on the General tab also *does* contain the macros and their settings.

The settings are stored in the json format.
You could theoretically modify this file by hand, but I am not sure if that is what you had in mind.

And sorry about the late reply.
